### PR TITLE
fix: update cloning and importing source book for testing

### PR DESCRIPTION
**Pull request by richard015ar**
Issue: #3008 

This PR replaces the `https://pbtest.pressbooks.com` testing book with a new testing book available at https://dev.pressbooks.pub/pbtest

A better approach for cloning and import routine is recommended: https://github.com/pressbooks/pressbooks/issues/3011

### CODE REVIEW

File: tests/test-integrations.php [hunk 1]
```diff
@@ -37,10 +37,10 @@ public function test_cloneRemoteBook() {
 		$this->assertNotEmpty( $meta );
 		$this->assertEquals( 'Public Domain', $meta['license']['name'] );
 
-		$cloned_items = $cloner->getClonedItems();
+		$cloned_items = $cloner->getClonedItems(); var_dump( $cloned_items );die();
```
Comment (SteelWagstaff) [inline tests/test-integrations.php:40]: I think we want to remove `var_dump` and `die` commands before PR is merged.

Comment (SteelWagstaff) [reply tests/test-integrations.php:40]:
```suggestion
		$cloned_items = $cloner->getClonedItems();
```

**Review by SteelWagstaff — APPROVED** [review]
LGTM. Thanks Ricardo!

**Comment by SteelWagstaff** [comment]
@richard015ar three tests still failing. Not sure why -- can we fix before merging?

**Comment by richard015ar** [comment]
I am taking a look right now.

**Comment by SteelWagstaff** [comment]
The problems have to do with the actual book we created at dev.pressbooks.pub. I've fixed a couple of problems. The ones I haven't fixed are connected to the WXR export/import. We can discuss at/after standup tomorrow. I think we're really close ...

**Comment by SteelWagstaff** [comment]
Ok -- I think have this working locally with a few changes I made and some changes to the book on pressbooks.pub. Going to push my fixes shortly.

**Comment by SteelWagstaff** [comment]
Tests are now passing for WP 6.0.3 but not WP latest (6.1). Not sure why. We can investigate more tomorrow ...

**Pull request merged**